cpp/ql/lib/semmle/code/cpp/pointsto/PointsTo.qll

```diff
@@ -19,6 +19,9 @@
  * `pointstoinfo` predicate determines the transitively implied points-to
  * information by collapsing pointers into equivalence classes. These
  * equivalence classes are called "points-to sets".
+ *
+ * WARNING: This library may perform poorly on very large projects.
+ * Consider using another library such as ??? instead.
```

Comment:
I think `DataFlow` should be able to solve most cases today.
```suggestion
 * Consider using another library such as `semmle.code.cpp.dataflow.DataFlow` instead.
```

Reply:
Sounds plausible, and its definitely better to have a suggestion that works for 80% than no suggestion at all.

Done.